students/piotrowski_stanislaw/lesson_03_functions/collatz_input_validation.py

```diff
@@ -0,0 +1,24 @@
+# https://automatetheboringstuff.com/chapter3/
+# "The Collatz Sequence + Input Validation"
+# piotrsta
+
+
+def collatz(number):
+    try:
+        if int(number) <= 0:
```

Comment:
konwersja do int to nie jest najlepsze rozwiązanie w tym miejscu, lepiej takie coś umieścić przy pobieraniu danych od użytkownika i dopiero jak jest poprawny int to uruchomić funkcje collatz